app/Domain/Integrations/Events/UdbOrganizerAdded.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Integrations\Events;
+
+use Illuminate\Foundation\Events\Dispatchable;
+use Ramsey\Uuid\UuidInterface;
+
+final readonly class UdbOrganizerAdded
+{
+    use Dispatchable;
+
+    public function __construct(public UuidInterface $id)
```

Comment:
I wonder if this should not contain the integration ID and the organizer ID? Because an UDB3 organizer is added to an integration, so to define the event uniquely both IDs are required.

Reply:
Renamed to `UdbOrganizerCreated`